Recrutify/Recrutify.Host/Controllers/SchedulesController.cs

```diff
@@ -1,11 +1,11 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
-using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
-using Recrutify.DataAccess.Models;
 using Recrutify.Services.DTOs;
+
```

Comment:
What is it? Do you need empty line and next using after it ?

Reply:
+